docs/guide/deployment/authorization-server.md

```diff
@@ -1,65 +1,158 @@
+| `api.grpc.address` | string | `listen specification <host>:<port> for grpc client connection.` | `"0.0.0.0:9100"` |
+| `api.grpc.tls.caPool` | string | `file path to the root certificates in PEM format` |  `""` |
+| `api.grpc.tls.keyFile` | string | `file name of private key in PEM format` | `""` |
+| `api.grpc.tls.certFile` | string | `file name of certificate in PEM format` | `""` |
+| `api.grpc.tls.clientCertificateRequired` | bool | `require client certificate` | `true` |
+
+### Authorization Client for Grpc Connectivity
+
+| Property | Type | Description | Default |
+| ---------- | -------- | -------------- | ------- |
+| `api.grpc.authorization.authority` | string | `endpoint of oauth provider` | `""` |
+| `api.grpc.authorization.audience` | string | `audience of oauth provider` | `""` |
+| `api.grpc.authorization.ownerClaim` | string | `owner claim of oauth provider` | `"sub"` |
+| `api.grpc.authorization.http.maxIdleConns` | int | `controls the maximum number of idle (keep-alive) connections across all hosts. Zero means no limit.` | `16` |
+| `api.grpc.authorization.http.maxConnsPerHost` | int | `optionally limits the total number of connections per host, including connections in the dialing, active, and idle states. On limit violation, dials will block. Zero means no limit.` | `32` |
+| `api.grpc.authorization.http.maxIdleConnsPerHost` | int | `if non-zero, controls the maximum idle (keep-alive) connections to keep per-host. If zero, DefaultMaxIdleConnsPerHost is used.` | `16` |
+| `api.grpc.authorization.http.idleConnTimeout` | string | `the maximum amount of time an idle (keep-alive) connection will remain idle before closing itself. Zero means no limit.` | `30s` |
+| `api.grpc.authorization.http.timeout` | string | `a time limit for requests made by this Client. A Timeout of zero means no timeout.` | `10s` |
+| `api.grpc.authorization.http.tls.caPool` | string | `file path to the root certificates in PEM format` |  `""` |
+| `api.grpc.authorization.http.tls.keyFile` | string | `file name of private key in PEM format` | `""` |
+| `api.grpc.authorization.http.tls.certFile` | string | `file name of certificate in PEM format` | `""` |
+| `api.grpc.authorization.http.tls.useSystemCAPool` | bool | `use system certification pool` | `false` |
+
+### Http Connectivity
+
+| Property | Type | Description | Default |
+| ---------- | -------- | -------------- | ------- |
+| `api.http.address` | string | `listen specification <host>:<port> for http client connection.` | `"0.0.0.0:9100"` |
+| `api.http.tls.caPool` | string | `file path to the root certificates in PEM format` |  `""` |
+| `api.http.tls.keyFile` | string | `file name of private key in PEM format` | `""` |
+| `api.http.tls.certFile` | string | `file name of certificate in PEM format` | `""` |
+| `api.http.tls.clientCertificateRequired` | bool | `require client certificate` | `true` |
+
+### Authorization Client for Device OAuth Provider
```

Comment:
`OAuth2.0 Client Configuration for Device`

Reply:
Add a note with explanation what is it used for:
`> Configured OAuth2.0 client is used to request an authorization code used for onboarding and exchange it for the token during the [cloud registration](https://plgd.dev/guide/architecture/component-overview.html#coap-gateway).`